tests/agent/test_prompt.py

```
@@ -0,0 +1,250 @@
+        )
+        print("Compilation successful!")
+
+        # Run verification tests
+        small_passed = False
+        large_passed = False
+
+        try:
+            verify_kernel_small_tensor(mod)
+            small_passed = True
+            print("Small tensor test passed")
+        except Exception as e:
```

Ruff (0.14.10)

Do not catch blind exception: `Exception`

(BLE001)